### DISCUSSION TITLE
From about v0.1.4, SSL error coming from ActualBudget in response to the cURL command

### DISCUSSION THREAD

**Discussion by hrpomrx** [bobokun/actualtap-py#98] [General]
`{"detail":"HTTPSConnectionPool(host='<actual_budget_url>', port=443): Max retries exceeded with url: /account/login (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)')))"}``

I am using Caddy; and have the root cert on my phone and mac; this seems to be a problem with actualtap talking to actual budget...have also tried adding the root/intermediate certs for Caddy onto the containers.

**Comment by bobokun** [comment]
Does this help? https://github.com/MattFaz/actualtap?tab=readme-ov-file#caddy
